Formula/php@7.3.rb

```diff
@@ -50,6 +50,11 @@ class PhpAT73 < Formula
   uses_from_macos "libxslt"
   uses_from_macos "zlib"
 
+  patch do
+    url "https://github.com/php/php-src/commit/aa405b7da270595d349d0596ad31305a41d4b1c0.patch"
```

Comment:
```suggestion
    url "https://github.com/php/php-src/commit/aa405b7da270595d349d0596ad31305a41d4b1c0.patch?full_index=1"
```

Reply:
Done. One of the github actions has already yelled at me about that. 😉